doc/classes/TabContainer.xml

```diff
@@ -118,7 +123,7 @@
 			<argument index="0" name="align" type="int" enum="TabContainer.TabAlign">
 			</argument>
 			<description>
-				Set tab alignment, from the ALIGN_* enum. Moves tabs to the left, right or center.
+				Set tab alignment, from the ALIGN_* enum. Moves tabs to the left, right, or center.
```

Comment:
- Set -> Sets
- I prefer calling them constants over enum, since that's how they're labeled on the page:

    Sets tab alignment using [code]ALIGN_*[/code] constants.

Reply:
Oh woops, didn't even notice it was saying "enum" here. I was making edits to a DOCS PR someone else had done previously.

Reply:
fixed